Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
```

Comment:
Why ancient LTS and not the last one 16.04. Having said that there are better Docker friendly variants of Ubuntu available. Have a look at https://github.com/phusion/baseimage-docker I use that for my OM docker

Reply:
Can I just switch without changing my commands?

Reply:
Sure. As I said. Look at my setup which you can also test.

Reply:
ok, I'll look into that